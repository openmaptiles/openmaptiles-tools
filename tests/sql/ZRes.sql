-- ZRes
SELECT round(ZRes(0)::numeric, 4) AS v1;	-- 156543.0339
SELECT round(ZRes(19)::numeric, 4) AS v2;	-- 0.2986
SELECT round(ZRes(0.5)::numeric, 4) AS v3;	-- 110692.6408
SELECT ZRes(NULL) AS null_v4;	-- \\N


