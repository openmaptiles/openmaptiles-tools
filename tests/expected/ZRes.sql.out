     v1      
-------------
 156543.0339
(1 row)

   v2   
--------
 0.2986
(1 row)

     v3      
-------------
 110692.6408
(1 row)

 null_v4 
---------
        
(1 row)

